docs/en/ingest-management/commands.asciidoc

```diff
@@ -834,6 +835,63 @@ elastic-agent install --url=https://fleet-server:8220 \
   --certificate-authorities=/path/to/ca.crt
 ----
 
+++++
+<hr>
+++++
+
+[discrete]
+[[elastic-agent-otel-command]]
+== elastic-agent otel
+
+preview::[]
+
+Run {agent} as an <<otel-agent,OpenTelemetry Collector>>. 
+
+[discrete]
+=== Synopsis
+
+[source,shell]
+----
+elastic-agent otel [flags]
+elastic-agent otel [command]
+----
+
+[discrete]
+=== Available commands
+
+`--validate`::
+Validates the OpenTelemetry collector configuration without running the collector.
+
+[discrete]
+=== Flags
+
+`--config --config=file:/path/to/first --config=file:path/to/second`::
+Locations to the config file(s). Note that only a single location can be set per flag entry, for example `--config=file:/path/to/first --config=file:path/to/second`.
+
+`--feature-gates flag`::
+Comma-delimited list of feature gate identifiers. Prefix with `-` to disable the feature. Prefixing with `+` or no prefix will enable the feature.
+
+`-h, --help`::
+Get help for the `otel` sub-command. Use `elastic-agent otel [command] --help` for more information about a command.
+
+`--set stringArray`::
+Set an arbitrary component config property. The component has to be defined in the configuration file and the flag has a higher precedence. Array configuration properties are overridden and maps are joined. For example, `--set=processors.batch.timeout=2s`.
```

Comment:
This notation works fine. Let me just note that this notation is not in line with upstream docs, which use the following notation that uses double colons `::`` instead of dots `.`:

```suggestion
Set an arbitrary component config property. The component has to be defined in the configuration file and the flag has a higher precedence. Array configuration properties are overridden and maps are joined. For example, `--set=processors::batch::timeout=2s`.
```

Not sure if this has any practical implications at the moment.

Reply:
Changed to:

![Screenshot 2024-06-18 at 1 26 33 PM](https://github.com/elastic/ingest-docs/assets/41695641/25340d34-ad2f-4f05-bc7c-57d78a24dca3)